en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md

```diff
@@ -0,0 +1,1060 @@
+# Amazon SQS Connector Reference
+
+The following operations allow you to work with the Amazon SQS Connector. Click an operation name to see parameter details and samples on how to use it.
+
+---
+
+## Initialize the connector
+
+To use the Amazon SQS connector, add the element in your configuration before carrying out any Amazon SQS operations. This Amazon SQS configuration authenticates with Amazon SQS by specifying the AWS access key ID and secret access key ID, which are used for every operation.
```

Comment:
Adding the init element was necessary in the older connector model. In this 2.x series we are using a connection and the config key. Shall we update this description accordingly?